manifests/istio-telemetry/kiali/templates/configmap.yaml

```diff
@@ -34,14 +34,10 @@ data:
         url_service_version: http://istio-pilot.{{ .Values.global.configNamespace }}:8080/version
       tracing:
         url: {{ .Values.kiali.dashboard.jaegerURL }}
-{{- if .Values.kiali.dashboard.inclusterjaegerURL }}
-        in_cluster_url: {{ .Values.kiali.dashboard.inclusterjaegerURL }}
-{{- end }}
+        in_cluster_url: {{ .Values.kiali.dashboard.jaegerInClusterURL }}
       grafana:
         url: {{ .Values.kiali.dashboard.grafanaURL }}
-{{- if .Values.kiali.dashboard.inclustergrafanaURL }}
-        in_cluster_url: {{ .Values.kiali.dashboard.inclustergrafanaURL }}
-{{- end }}
```

Comment:
don't we still want these if..end statements? Its possible the user set those settings to "" explicitly (perhaps because they don't have them installed).